docs/tutorials/schedule-jobs.md

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_position: 5
+---
+
+# Submit your job
+
+Are you ready to deploy your plugins to measure the world? We will use [edge scheduler](../about/architecture.md#edge-scheduler-es) to submit a job to demonstrate how you can deploy plugins to field-deployed Waggle nodes.
+
+:::caution
+If you have not created your account, please go to [here](https://access.sagecontinuum.org) and log in to create a new account with your email. Once logged in, you will be able to create and edit your jobs, but will need a permission to submit jobs to the scheduler. Please contact us at [contact-us](../contact-us.md) to request the permission for job submission.
+:::
+
+A job describes,
+- plugins that are registered and built in [edge code repository](../about/architecture.md#edge-code-repository-ecr) with specification including any plugin arguments,
+- a list of Waggle nodes on which the plugins will be run,
+- science rules describing when the plugins should be scheduled,
+- conditions to determine when the job completes
+
+## Create a job
+
+We create a job in YAML format (JSON format is also supported. Please check out details of job attributes [here](https://github.com/waggle-sensor/edge-scheduler/tree/main/docs/sesctl).)
+
+```bash
+cat << EOF > myjob.yaml
+---
+name: myjob
+plugins:
+- name: image-sampler
+  pluginSpec:
+    image: registry.sagecontinuum.org/theone/imagesampler:0.3.0
+    args:
+    - -stream
+    - bottom
+nodes:
+  W030:
+scienceRules:
+- "image-sampler: cronjob('image-sampler', '* * * * *')"
+successcriteria:
+- WallClock(1d)
+EOF
+```
+
+We attempt to run a plugin named `image-sampler` to collect an image from the camera named `bottom` on `W030` node. As a result of the run, we will get images from the job. The job also specifies that the plugin needs to be scheduled every minute (i.e., `* * * * *` in [crontab expression](https://crontab.guru/)). The job completes 24 hours after the job started to run on the node.
+
+## Submit the job using _sesctl_
+
+`sesctl` is a command-line tool to manage jobs exist in the scheduler.
```

Comment:
How do they get this tool on their machine?  link to the github project where they can clone / access it?

Reply:
Done